### ISSUE TITLE
Help with installation

### ISSUE THREAD

**Issue by rastacalavera** [mattstratton/castanet#340]
Hello there!
I am really interested in trying out your theme but for the life of me I can't get it to launch. 
The link to installation help in the readme just directs to the main hugo page which isn't really helpful. 
I've made sure to install the latest hugo and go (not from ubuntu repos) and have downloaded the latest version of castanet.
I copied the contents from the example site into the folder structure that is created when running ```hugo new site``` and then i get the following error ```Error: module "castanet" not found; either add it as a Hugo Module or store it in "/home/rastacalavera".: module does not exist``` 

I tried following the module documentation from Hugo and couldn't get that to work either. I've never used modules before so that might be the issue (my inexperience). 

Any help would be very much appreciated, this seems like an awesome theme!

So just for information, here is my current structure:
~/git/podcast --> where i had hugo create a new site and copied over the contents from the exampleSite
~/git/podcast/themes --> contains castanet-main folder

I can provide any needed additional information.

**Comment by mattstratton** [comment]
I'm wondering if there is something in your config that is making it want to look for a module? 

any chance you can share what your `config.toml` looks like?

**Comment by rastacalavera** [comment]
Thanks for the response! I think I was trying a few too many things to get it running. I played with the [arrested dev ops theme](https://github.com/arresteddevops/ado-hugo) and got my bearings there, then came back to this today.
I think what I may have been doing was moving all the files out of the exampleSite into the main castanet folder and then trying to run ```hugo server``` from there. Or else it was something else. . . i typically hit several snags and just tried to solve each individual error rather than just going back and looking more closely at your documentation.
With fresh eyes I looked at the README.md and followed the link to the [contributing guide](https://github.com/mattstratton/castanet/blob/main/CONTRIBUTING.md) which clearly laid out how to properly setup the environment for testing.
I now have it up and running and am playing with it!
